CHANGELOG.md

```diff
@@ -45,10 +45,19 @@ The format is based on [Keep a Changelog].
     | `M-s p` | `selectrum-prescient-toggle-prefix`     |
     | `M-s r` | `selectrum-prescient-toggle-regexp`     |
 
+* The custom variable `prescient-filter-alist` was added, which
+  describes the relationship between the symbols in
+  `prescient-filter-list` and the corresponding functions that produce
+  regular expressions for matching candidates. Users can create their
+  own filter methods by adding a symbol-function pair to
+  `prescient-filter-alist` and use that custom method by adding the
+  symbol to `prescient-filter-method`. See [#77].
```

Comment:
Should this mention `selectrum-prescient-create-and-bind-toggle-command` as well, since it's a new public function?

Reply:
I've updated the README to mention it under the list item which describes the toggles.